robot_skills/src/robot_skills/get_robot.py

```diff
@@ -4,19 +4,29 @@
 # ROS
 import rospy
 
-from .amigo import Amigo
-from .hero import Hero
-from .mockbot import Mockbot
 # Robot skills
 from .robot import Robot
-from .sergio import Sergio
 
-ROBOTS = {
-    "amigo": Amigo,
-    "hero": Hero,
-    "mockbot": Mockbot,
-    "sergio": Sergio
-}
+# ToDo: create a decent registration here
```

Comment:
Create an issue for this

Reply:
See https://github.com/tue-robotics/tue_robocup/issues/1095